gcp-test/tests/gcp_dns_record_set/test-turbot-expected.json

```diff
@@ -0,0 +1,6 @@
+[
+	{
+	  "akas": ["{{ output.resource_aka.value }}"],
+	  "title": "{{ output.record_set_name.value }}"
+	}
```

Comment:
@ParthaI Can you please check you vscode settings file?
Seems like it is using 4 spaces

Reply:
Modified the settings to 2 spaces

Reply:
@ParthaI Can you please check the indentation again?
Seems like still it is using 4 spaces